web/packages/teleport/src/services/userEvent/types.ts

```diff
@@ -85,9 +85,7 @@ export enum DiscoverEvent {
   DatabaseRegister = 'tp.ui.discover.database.register',
   DatabaseConfigureMTLS = 'tp.ui.discover.database.configure.mtls',
   DatabaseConfigureIAMPolicy = 'tp.ui.discover.database.configure.iampolicy',
-  DesktopActiveDirectoryToolsInstall = 'tp.ui.discover.desktop.activeDirectory.tools.install',
   DesktopActiveDirectoryConfigure = 'tp.ui.discover.desktop.activeDirectory.configure',
-  AutoDiscoveredResources = 'tp.ui.discover.autoDiscoveredResources',
```

Comment:
i'd put this back, it wasn't made specifically for desktops (though it was the only resource emitting it), but it maybe useful later?

Reply:
I think we should just bring it back if/when we actually need it ([YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)). It looks like I missed the `DesktopActiveDirectoryConfigure` above this, too.